test/e2e/framework/authorizer_util.go

```diff
@@ -63,3 +68,47 @@ func WaitForAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviewsGette
 	})
 	return err
 }
+
+// BindClusterRole binds the cluster role at the cluster scope
+func BindClusterRole(c v1alpha1rbac.ClusterRoleBindingsGetter, clusterRole, testNamespace string, subjects ...rbacv1alpha1.Subject) {
+	// this test wants powerful permissions.  Since the namespace names are unique, we can leave this
+	// lying around so we don't have to race any caches
+	_, err := c.ClusterRoleBindings().Create(&rbacv1alpha1.ClusterRoleBinding{
+		ObjectMeta: legacyv1.ObjectMeta{
+			Name: testNamespace + "--" + clusterRole,
+		},
+		RoleRef: rbacv1alpha1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     clusterRole,
+		},
+		Subjects: subjects,
+	})
+
+	// if we failed, don't fail the entire test because it may still work. RBAC may simply be disabled.
+	if err != nil {
+		fmt.Printf("Error binding clusterrole/%s for %q for %v\n", clusterRole, testNamespace, subjects)
+	}
+}
+
+// BindClusterRoleInNamespace binds the cluster role at the namespace scope
+func BindClusterRoleInNamespace(c v1alpha1rbac.RoleBindingsGetter, clusterRole, testNamespace string, subjects ...rbacv1alpha1.Subject) {
+	// this test wants powerful permissions.  Since the namespace names are unique, we can leave this
+	// lying around so we don't have to race any caches
+	_, err := c.RoleBindings(testNamespace).Create(&rbacv1alpha1.RoleBinding{
+		ObjectMeta: legacyv1.ObjectMeta{
+			Name: testNamespace + "--" + clusterRole,
+		},
+		RoleRef: rbacv1alpha1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     clusterRole,
+		},
+		Subjects: subjects,
+	})
+
+	// if we failed, don't fail the entire test because it may still work. RBAC may simply be disabled.
+	if err != nil {
+		fmt.Printf("Error binding clusterrole/%s into %q for %v\n", clusterRole, testNamespace, subjects)
```

Comment:
> In the old code we checked for apierrors.IsForbidden(err). What do we get if rbac is disabled?

There are various failure modes.  403 if they're just missing permissions  404 if they have permissions and its not enabled.  In the end, I don't think it much matters.  If it fails and there's not a permission via a webhook or something, this will just fail.